app/pathogen/arbovirus/analyze/recharts.tsx

```diff
@@ -240,8 +240,27 @@ interface WHORegionAndArbovirusData extends dataStratifiedByArbovirus {
   region: string;
 }
 
-export function WHORegionAndArbovirusBar() {
+export enum LegendConfiguration {
+  RIGHT_ALIGNED = 'RIGHT_ALIGNED',
+  BOTTOM_ALIGNED = 'BOTTOM_ALIGNED'
+}
+
+interface WhoRegionAndArbovirusBarInput {
+  legendConfiguration: LegendConfiguration;
+}
+
+export function WHORegionAndArbovirusBar(input: WhoRegionAndArbovirusBarInput) {
   const state = useContext(ArboContext);
+  const legendProps = input.legendConfiguration === LegendConfiguration.RIGHT_ALIGNED ? {
+    layout: "vertical" as const,
```

Comment:
`as const`s are necessary here to narrow the type down instead of it going to `string`.